simple_certmanager/models.py

```diff
@@ -98,3 +116,35 @@ def is_valid_key_pair(self) -> None | bool:
         return key_pubkey == cert_pubkey
 
     is_valid_key_pair.boolean = True  # type: ignore
+
+
+class CertificateSigningRequestInfo(models.Model):
+    country_name = models.CharField(
+        _("country name"),
+        max_length=2,
+        help_text=_("Two-letter country code"),
+        default="NL",
+    )
+    organization_name = models.CharField(
+        _("organization name"),
+        max_length=100,
+        help_text=_("The name of the organization, e.g. 'Gementee Amsterdam'"),
```

Comment:
some organizations will not appreciate that "Amsterdam" is here, let's use a fictive organization name:

```suggestion
        help_text=_("The name of the organization, e.g. 'OpenGem'"),
```